maps.md

```diff
@@ -566,8 +564,6 @@ func (d Dictionary) Update(word, definition string) error {
 	default:
 		return err
 	}
-
-	return nil
```

Comment:
Like my previous comment, I don't think this code is really unreachable, since this return is hit when there are no errors :confused: